tests/test_matching.py

```diff
@@ -12,6 +12,18 @@
 from selenium.common.exceptions import TimeoutException
 from selenium.common.exceptions import NoSuchElementException
 
+def _matching_invitation_ids(client, group_id):
```

Comment:
We don't need this function anymore, right?

Reply:
did you want me to convert *all* cases where this is used into explicit strings? or just the assertion lines?

Reply:
I think you are using this in all the assertion lines.